ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/mac-security.j2

```diff
@@ -0,0 +1,52 @@
+{% if mac_security is defined and mac_security is not none %}
+### MACsec Summary
+
+{%     if mac_security.license is defined and mac_security.license is not none %}
+License installed!
+{%   else %}
+License missing!
```

Comment:
Depending on the image type this may not be a problem. I suggest changing this to `License not installed!`

Reply:
Updated.